source/opt/strength_reduction_pass.cpp

```diff
@@ -164,6 +164,10 @@ uint32_t StrengthReductionPass::GetConstantId(uint32_t val) {
         context(), SpvOp::SpvOpConstant, uint32_type_id_, resultId, {constant}));
     get_module()->AddGlobalValue(std::move(newConstant));
 
+    // Notify the DefUseManager about this constant.
```

Comment:
Good catch

Reply:
We should probably call the IRContext's version of AddGlobalValue, and have that update the def-use manager.

Reply:
I filed #1000 for that.